CONTRIBUTING.md

```diff
@@ -1,23 +1,57 @@
-  * [Answer questions on Stack Overflow](http://stackoverflow.com/questions/tagged/video.js)
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Other repositories where issues could be filed](#other-repositories-where-issues-could-be-filed)
+- [Filing issues](#filing-issues)
+  - [Reporting a Bug](#reporting-a-bug)
+  - [Requesting a Feature](#requesting-a-feature)
+- [Contributing code](#contributing-code)
+  - [Building video.js locally](#building-videojs-locally)
+    - [Forking and cloning the repository](#forking-and-cloning-the-repository)
+    - [Installing local dependencies](#installing-local-dependencies)
+    - [Running tests](#running-tests)
+    - [Building videojs](#building-videojs)
+    - [Testing Locally](#testing-locally)
+    - [Sandbox test directory](#sandbox-test-directory)
+    - [Running a local web server](#running-a-local-web-server)
+    - [Watching source and test changes](#watching-source-and-test-changes)
+  - [Making Changes](#making-changes)
+    - [Step 1: Verify](#step-1-verify)
+    - [Step 2: Update remote](#step-2-update-remote)
+    - [Step 3: Branch](#step-3-branch)
+    - [Step 4: Commit](#step-4-commit)
+    - [Step 5: Test](#step-5-test)
+    - [Step 6: Push](#step-6-push)
+  - [Code Style Guide](#code-style-guide)
+- [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+- [Doc Credit](#doc-credit)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Other repositories where issues could be filed
 
 There's also other Video.js projects where you can help. (check the [video.js org](https://github.com/videojs) for an up-to-date list of projects)
 
   * [Videojs.com](https://github.com/videojs/videojs.com)
-  * [Video.js flash player](https://github.com/videojs/video-js-swf)
-  * [Player skin designer](https://github.com/videojs/designer)
-  * [Contribflow](https://github.com/zencoder/contribflow)
+  * [Video.js flash player](https://github.com/videojs/videojs-flash)
+  * [HLS](https://github.com/videojs/videojs-contrib-hls)
+  * [DASH](https://github.com/videojs/videojs-contrib-dash)
+  * [Youtube Tech](https://github.com/videojs/videojs-youtube)
+  * [Vimeo Tech](https://github.com/videojs/videojs-vimeo)
+  * [Ads](https://github.com/videojs/videojs-contrib-ads)
+  * [Plugin generator](https://github.com/videojs/generator-videojs-plugin)
+  * [Linter][linter]
+
+## Filing issues
 
-Filing issues
--------------
 [GitHub Issues](https://github.com/videojs/video.js/issues) are used for all discussions around the codebase, including **bugs**, **features**, and other **enhancements**.
 
+When filling out an issue, make sure to fill out the questions in the 
 ### Reporting a Bug
 
 **A bug is a demonstrable problem** that is caused by the code in the repository. Good bug reports are extremely helpful. Thank You!
```

Comment:
Maybe add a more explicit note that bugs in plugins should be filed against those plugins.